cabal-testsuite/PackageTests/Backpack/T5634/setup.test.hs

```diff
@@ -1,5 +1,8 @@
 import Test.Cabal.Prelude
-main = setupAndCabalTest $ do
+main = do
+  -- TODO: this might be a GHC bug that needs fixing
+  skipIfAlpine "bug #11041"
```

Comment:
TBH I feel like this should be in a separate PR. I can cut it and get it in if you'd prefer not to be juggling another one.

Reply:
If this is removed, then the PR CI fails, so I disagree that this needs to be in another PR.